lib/3scale_toolbox/commands/import_command/openapi/mapping_rule.rb

```diff
@@ -0,0 +1,35 @@
+module ThreeScaleToolbox
+  module Commands
+    module ImportCommand
+      module OpenAPI
+        module MappingRule
+
+          def mapping_rule
+            {
+              'pattern' => pattern,
+              'http_method' => http_method,
+              'delta' => delta,
+              'metric_id' => metric_id
+            }
+          end
+
+          def http_method 
+            operation[:verb].upcase
+          end
+
+          def pattern
+            operation[:path]
```

Comment:
Is the pattern compatible with what mapping rules support?

Reply:
haven't checked that. Is it documented what mapping rules support in `pattern` field? Send link please.

Reply:
The documentation: https://github.com/3scale/porta/blob/8022f744c58a8a64f6fcfb8d2f2129fd9eaad74d/config/locales/en.yml#L530-L557

The implementation: https://github.com/3scale/apicast/blob/c3e431df20f65686798d1e67d8c4e9f568ef9c08/gateway/src/apicast/mapping_rule.lua#L30-L35

Reply:
Mapping rules accepted expression set includes openapi 2.0 path expressions.

The field name MUST begin with a slash and Path Templating is allowed.
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#patterned-fields

Reply:
I'm not sure it works. 

Defining a path `/pets` would match `/pets/{id}` too. Is that desired? Our expressions are not the exact match to the whole string, just matching the beginning.

Reply:
Cannot find path matching spec in OAS 2.0 spec. I just found [clarifications on path overlapping issues](https://github.com/OAI/OpenAPI-Specification/issues/1127)

From common sense, I would expect OAS 2.0 spec to do exact match. Just a guess.

Since apicast does just prefix matching, I guess we need to specify somehow end of string when creating mapping rule in 3scale. That only if my assumption about exact match on OAS 2.0 is correct. Is there a way to do that? Adding '/' in the end would not work.

Reply:
@eguzki it is described in the documentation mentioned in https://github.com/3scale/3scale_toolbox/pull/76#discussion_r241044806 (using $ at the end)